config-sample.yaml

```diff
@@ -60,6 +60,9 @@ keys:
   key1: secret1
   key2: secret2
 
+auth_type: redis
```

Comment:
```suggestion
# # auth details could be stored in Redis. when enabled, ....
# key_provider: redis
```
once we decide the redis structure, let's include notes on how they keys should be stored in redis.